action.yml

```diff
@@ -18,7 +18,7 @@ inputs:
       general recommendation is to pin to exact tags or commit SHAs.
 
 
-      Please also consider migrading your setup to use secretless publishing:
+      Please also consider migrating your setup to use secretless publishing:
```

Comment:
@hugovk could you also fix the same sentence below?

Reply:
Done!